### ISSUE TITLE
Global hotkeys not working in games

### ISSUE THREAD

**Issue by EDZ1LLA** [aleab/toastify#163]
If this is a crash, bug or unexpected behaviour report, please use the following template.

## EXPECTED BEHAVIOUR
Global hotkeys are pressed in game (LoL) and the corresponding action is taken.

## ACTUAL BEHAVIOUR
Global hotkey pressed and is not registered or acted upon by toastify.

## STEPS TO REPRODUCE THE BEHAVIOUR
Set up global hotkeys in toastify. Press them when in game in LoL (or other fullscreen game).

**Comment by Shqcky** [comment]
I still have this issue but only in lol, did you find a way to fix it?

**Comment by EDZ1LLA** [comment]
Not Really. Its the anti cheat in lots of games that stop it working. A potential solution was to use autohotkeys to trigger the actions of toastify. There was a few posts about how to configure autohotkeys to work in different games with extended presses etc to make sure they trigger. It seemed to work for people but it was a step I decided ultimately wasn't worth it for me.
